utils/fs/os/os.go

```diff
@@ -21,7 +21,8 @@ func NewOS(baseDir string) *OS {
 	}
 }
 
-// Create creates a new GlusterFSFile
+// Create creates a file and opens it with standard permissions
+// and modes O_RDWR, O_CREATE and O_TRUNC.
 func (fs *OS) Create(filename string) (File, error) {
```

Comment:
to avoid duplicate code you can call `fs.OpenFile(filename, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)`, and the same for Open